packages/api-docs-builder/buildApi.ts

```diff
@@ -119,6 +124,7 @@ async function buildSingleProject(
     mkdirSync(manifestDir, { recursive: true });
   }
 
+  rawDescriptions = Boolean(projectSettings.rawDescriptions);
```

Comment:
That feels a bit weird, because reding the settings in the async function has side effect on other the imported functions. So I'm unsure about what happends if on project settings is set to `true` and the other is `undefined`. Could it suffer race condition?

What about moving the `rawDescriptions` from project level to script level by putting it in the `buildApi`

```
buildApi(projects, grep = null, rawDescription = false)

pnpm docs:api --rawDescription
```

Reply:
I wondered the same at first, but all calls to the async functions in this package are awaited, so the code is effectively run synchronously. I also have tried it with different projects having `rawDescriptions` setting and haven't seen anything unexpected.

DX-wise it makes more sense on the project settings to me but I don't feel strongly either way, so it's your call. What would you prefer to settle on here?